easybuild/framework/easyblock.py

```diff
@@ -3739,16 +3739,16 @@ def xs2str(xs):
         if self.toolchain.mpi_family() and self.toolchain.mpi_family() in toolchain.OPENMPI:
             env.setvar('OMPI_MCA_rmaps_base_oversubscribe', '1')
 
-        # change to install directory (better environment for running tests)
-        if os.path.isdir(self.installdir):
-            change_dir(self.installdir)
+        # run sanity checks from an empty temp directory
+        # using the build or installation directory can produce false positives and polute them with files
+        sanity_work_dir = tempfile.mkdtemp(prefix='eb-saniy-check-')
```

Comment:
@lexming Are you testing whether or not I'll catch the typo here? ;)

Reply:
ohh good one :smile:

Reply:
fixed in 2389fc6